lib/Doctrine/ORM/Tools/SchemaValidator.php

```diff
@@ -118,7 +121,7 @@ public function validateClass(ClassMetadata $class): array
         }
 
         // PHP 7.4 introduces the ability to type properties, so we can't validate them in previous versions
-        if (PHP_VERSION_ID >= 70400) {
+        if (PHP_VERSION_ID >= 70400 && $this->validatePropertyTypes) {
```

Comment:
I will create another PR after this one is merged to remove the `PHP_VERSION_ID` here and in other places.